wordpress-mobile/WordPress-iOS
# Issues/aztec media progress report

This PR adds progress reporting while an upload of a media object is happening.

It show progress for each image on top of it and global progress on the navigation bar.

To test:
 - Activate Aztec editor
 - Create a new post
 - Add one image/ multiple images from different sources

Check if the progress indicator show up correctly.

Needs review: @jleandroperez
